dist/goldRush.js

```diff
@@ -0,0 +1,180 @@
+/*
+Author: Anas Nasrallah.
+Peupose: Gold Rush Game.
+Date: 30.05.20
+*/
+
+const EMPTY_CELL = '.';
+const COIN_CELL = 'c';
+const BLOCK_CELL = 'b'
+const BOOST_CELL = 'boost'
+const P1_CELL = 1;
+const P2_CELL = 2;
+const SCORE_PER_COIN = 10;
+const ADDITION_BOOSTS = 3;
+const RIGHT = 'right'
+const LEFT = 'left'
+const DOWN = 'down'
+const UP = 'up'
+
+class GoldRush extends Matrix {
+    constructor(r, c) {
+        super(r, c)
+        this.p1 = { sign: 1, x: 0, y: 0, score: 0, boosted: 0 }
+        this.p2 = { sign: 2, x: c - 1, y: r - 1, score: 0, boosted: 0 }
```

Comment:
You should use clear and understandable names for classes, functions, and variables (for example: player1 etc.) and not something general like "p1", "str" etc.